screens/ShoppingList.js

```diff
@@ -0,0 +1,114 @@
+
+import React, { Component } from 'react';
+import { StyleSheet, Text, View, Dimensions, Image, AppRegistry } from 'react-native';
+
+export default class ShoppingList extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={styles.orangeBox}>
+          <View style={styles.formatInOrangeBox}>
+            <Image source ={require('./../assets/images/shopping-cart.png')} style={styles.ShoppingCart} />
+            <Image source ={require('./../assets/images/person.png')} style={styles.person} />  
+          </View>
+          <View style={styles.formatInOrangeBox}>
+            <Text style = {styles.header}>Shopping List</Text>
+          </View>
+        </View>
+        <View style={{flex: 1, justifyContent: 'flex-end',  alignItems: 'center'}}>
```

Comment:
Since there will be multiple results you should use a `FlatList`. The user will have to scroll through the results - right now you have hardcoded in views which isn't dynamic